app/src/main/res/layout/item_notification.xml

```diff
@@ -14,7 +14,6 @@
         android:layout_marginStart="16dp"
         android:layout_marginTop="16dp"
         android:background="@android:color/white"
-        android:contentDescription="@string/no_image_found"
```

Comment:
Should we add a more appropriate `contentDescription` instead? I remember these being added by someone while fixing lint issues.

Reply:
We could, but didn't think it was really necessary (especially with extra work translating etc.). All it'd be would be something like 'notification icon', which would be the case for all the notifications in the list. Similarly with the other change they'd all be 'place icon' or similar.